### PR TITLE
convert-for-checkers: share+removeUselessShares, smv: remove finals o…

### PR DESCRIPTION
**Pull request by p4l1ly**
…nly if necessary

### CODE REVIEW

File: convert-for-checkers.sh [hunk 1]
```diff
@@ -33,33 +34,62 @@ fi | while read -r fAfa; do
...
   ${Afasat:-false} && {
     echo "Transforming to .afasat"
-    $LTLE_TO_AFA removeFinalsNonsep < $f.afa | $LTLE_TO_AFA prettyToAfasat > $f.afasat
+    $AUTOMATA_SAFA_ONE share < $f.afa | $AUTOMATA_SAFA_ONE removeUselessShares | $AUTOMATA_SAFA_ONE removeFinals | $LTLE_TO_AFA prettyToAfasat > $f.afasat
   }
   ${Ada:-false} && {
     echo "Transforming to .ada"
     $LTLE_TO_AFA prettyToAda < $f.afa > $f.ada 2> /dev/null || \
-    $LTLE_TO_AFA removeFinalsNonsep < $f.afa | $LTLE_TO_AFA prettyToAda > $f.ada
+    $AUTOMATA_SAFA_ONE removeFinalsHind < $f.afa | $LTLE_TO_AFA prettyToAda > $f.ada
```
Comment (jurajsic) [inline convert-for-checkers.sh]: Why did you change this to `removeFinalsHind`? Shouldn't it be `$AUTOMATA_SAFA_ONE removeFinals`

Comment (p4l1ly) [reply convert-for-checkers.sh]: yes, thx

Comment (p4l1ly) [reply convert-for-checkers.sh]: hind is only good for separated automata (and keeps them separated)
